### DISCUSSION TITLE
When using a prefix in a route my layout loses styles

### DISCUSSION THREAD

**Discussion by Osorio95** [adonisjs/core#3629] [Help] [answered]
In this project I'm trying to deploy categories, like driver/main or user/main, differents pages, differents folders and the same @layout, when I try to use for expample driver-main instead driver/main the styles works, but with prefix they don't.

I'm using the simple public storage without webpack since I don't know how to use it.

**Comment by RomainLanz** [comment, marked as answer]
Hey @Osorio95! 👋🏻 

Could you give us some code examples? The best would be a reproducible repository.
Also, ensure to use an absolute path to define your asset.

**Reply by Osorio95** [reply]
Sure thing:

My route uses both cases

```
import Route from '"@"ioc:Adonis/Core/Route'

Route.get('/', async ({ view }) => {
  return view.render('index')
})

Route.get('/user/home', async ({ view }) => {
      return view.render('pages/passenger/main')
    })
```

My layouts/main.edge it's a bit extense, mainly it has my stylesheets, my scripts, the sidebar, navbar and the `@section('body')`.

The bodys are located in:
```
views/index.edge
views/pages/passengers/main.edge
```
the bodys are

for index:
```
@layout('layouts/main')
@set('title', 'Inicio')

@section('body')
      <section class="content-header">
        <div class="container-fluid">
          <div class="row mb-2">
            <div class="col-sm-6">
              <h1>Bienvenido a Home</h1>
            </div>
            <div class="col-sm-6">
              <ol class="breadcrumb float-sm-right">
                <li class="breadcrumb-item active">{{title}}</li>
              </ol>
            </div>
          </div>
        </div><!-- /.container-fluid -->
      </section>
@end
```

For the passengers/main:

```
@layout('layouts/main')
@set('title', 'Pasajeros')

@section('body')
      <section class="content-header">
        <div class="container-fluid">
          <div class="row mb-2">
            <div class="col-sm-6">
              <h1>{{title}}</h1>
            </div>
            <div class="col-sm-6">
              <ol class="breadcrumb float-sm-right">
                <li class="breadcrumb-item"><a href="{{ route('/') }}">Inicio</a></li>
                <li class="breadcrumb-item active">{{title}}</li>
              </ol>
            </div>
          </div>
        </div>
      </section>
@end
```

This happens only when I try to add a prefix for some pages, based on this example I have passengers/main which is not working, however if I do not index it, it works perfectly with passengers-main, passengers-page2, passengers-page3.

One thing I have to metion is that I don't have any controller set up.

**Reply by guntribam** [reply]
please [format ](https://docs.github.com/pt/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) your code ... we're trying to help you, but you can help us to help you in the firstplace

**Reply by Osorio95** [reply]
I'm sorry, fixed, thank you.

**Reply by Daghsny** [reply]
please how can i resolve the problem, i have the same one!

**Comment by nparoski** [comment]
In my case the issue was loading resources without `/` and the beginning.
e.g
```html
  <script src="vendor/jquery/jquery.min.js"></script>
```

instead of 

```html
  <script src="/vendor/jquery/jquery.min.js"></script>
```
